src/main/charts/bitbucket/templates/_helpers.tpl

```diff
@@ -41,15 +41,15 @@ else if we're creating a new service account then use the name of the Helm relea
 else just use the "default" service account.
 */}}
 {{- define "bitbucket.serviceAccountName" -}}
```

Comment:
The new unit tests found a bug in this template. Fixed it with the indiscriminate use of whitespace chomping indicators.